source/drivers/php-libraries.txt

```diff
@@ -37,6 +37,8 @@ Stand-alone Libraries
   **mongodb** extension. It is schema-less and supports fast object hydration
   and persistence, which makes it well-suited for modeling aggregation results.
 
+- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`_: Easy, powerful and ultrafast ODM for PHP and MongoDB.
```

Comment:
```suggestion
- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`_: Easy, powerful,
   and ultrafast ODM for PHP and MongoDB. Implements both ActiveRecord and
   DataMapper design patterns.
```

Fixing word wrap, Oxford comma, and adding another sentence to distinguish this library from other entries.